### ISSUE TITLE
babel-types: ClassMethod is missing static field

### ISSUE THREAD

**Issue by 43081j** [babel/babel#8607]
As you can see [here](https://github.com/babel/babel/blob/2921f6cdb1b9dede37a59231a18f50f6b53eae56/packages/babel-types/src/definitions/es2015.js#L390) we have a `static` field on methods/properties.

However, because `isValidIdentifier` will be false for this (reserved keyword), it is skipped when generating types.

Syntactically, I don't think TS will complain if we output:

```ts
interface Foo {
  static: boolean;
}
```

Though maybe if we encounter an "invalid identifier", we should just wrap it in quotes or some such thing?

**Comment by nicolo-ribaudo** [comment]
Do you want to open a PR?

**Comment by 43081j** [comment]
If we're sure that would be an appropriate way to handle invalid identifiers, sure.

I'll have a look into it.

**Comment by nicolo-ribaudo** [comment]
Yeah, quoting them is ok. Tecnically `static` an be unqoted while `foo-bar` can't: if there is an easy way to differentiate between them, lets not quote `static`. If it is too complex, we can quote both.
